Cargo.toml

```diff
@@ -8,6 +8,11 @@ homepage = "https://www.neon-bindings.com"
 repository = "https://github.com/neon-bindings/neon"
 license = "MIT/Apache-2.0"
 exclude = ["neon.jpg"]
+build = "build.rs"
+
+# This name is arbitrary, but allows us to re-export the location of node.lib
+# to dependent packages so they can link on Windows.
+links = "neon-sys"
```

Comment:
If this name is arbitrary, and we're hapy with the one chosen, maybe we could re-phrase this comment and maybe a link to the crates.io docs about this key.

```toml
# This name allows us to re-export the location of node.lib
# to dependent packages so they can link on Windows.
# http://doc.crates.io/build-script.html#the-links-manifest-key
```